### PR TITLE
TS: Added InstancedMesh.d.ts.

### PR DESCRIPTION
**Pull request by Mugen87**
see #17621

### CODE REVIEW

File: src/objects/InstancedMesh.js [hunk 1]
```diff
@@ -13,6 +13,8 @@ function InstancedMesh( geometry, material, count ) {
 
 	this.count = count;
 
+	this.type = 'InstancedMesh';
```
Comment (mrdoob) [inline src/objects/InstancedMesh.js]: Hmm, what's this for?

Comment (Mugen87) [reply src/objects/InstancedMesh.js]: I think we need this for serialization/deserialization. Adding support for `InstancedMesh` in `ObjectLoader` was my next task^^

Comment (Mugen87) [reply src/objects/InstancedMesh.js]: Wait! I've realized while working on this that `type` is not needed^^. Removing it here: 00e137184b23fe681eebbe263c879dee8aaecd01

Comment (mrdoob) [reply src/objects/InstancedMesh.js]: Thanks!

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**